### PR TITLE
[18223] Documentation: v0.1.0 release notes

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**

### CODE REVIEW

File: docs/rst/notes/notes.rst [hunk 1]
```diff
@@ -1,4 +1,33 @@
...
+2. Implementation of several `set` functions:
+    - DomainParticipant name.
+    - DomainParticipant external locators (default and metatraffic): externality, address, and port.
+    - DomainParticipant initial peers: address and port.
+    - DomainParticipant lease duration and lease announcements.
+    - DomainParticipant use of custom transports disabling builtin transports.
+    - Reliability and Durability QoS for DataReader and DataWriter profiles.
+    - Default profile attribute for DomainParticipant, DataReader and DataWriter.
+
+Fast DDS QoS Profiles Manager CLI
```
Comment (JesusPoderoso) [inline docs/rst/notes/notes.rst:21]: Include Transport descriptor `set` methods:
```suggestion
    - DomainParticipant use of custom transports disabling builtin transports.
    - Reliability and Durability QoS for DataReader and DataWriter profiles.
    - Default profile attribute for DomainParticipant, DataReader and DataWriter.
    - Transport descriptor kind.
    - Transport descriptor interface whitelist.

Fast DDS QoS Profiles Manager CLI
```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]
May the suggestion be applied also in the GitHub release notes?

File: docs/rst/notes/notes.rst (hunk 1 above)
Comment (JLBuenoLopez-eProsima) [reply docs/rst/notes/notes.rst:21]: Yep. I will modify the release notes in Github too. Thanks!

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**